### PR TITLE
Fix typo (github>GitHub)

### PR DESCRIPTION
**Pull request by himanshu3w**
Changed github typo to GitHub
@wackerow Take a look Please

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Approved - will merge after we merge deploy 👍

**Comment by himanshu3w** [comment]
Sure

**Pull request merged**
